client/src/partials/cash/modals/receipt.modal.js

```diff
@@ -29,6 +30,11 @@ function CashReceiptModalController(uuid, patientUuid, ModalInstance, Cash, Proj
     vm.error = error;
   }
 
+  /** @todo implement a good way print function */
+  function handlePrint() {
```

Comment:
In the future, you can use `angular.noop()` to indicate a function that is unimplemented.  It helps developers know that you purposely left something undone, rather than simply an error :)